.github/workflows/athena_test.yml

```
@@ -64,6 +66,17 @@ jobs:
           if [ "$overall_success" = false ]; then
             exit 1
           fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)